File: OtherComponents/MDWSvistalayer/MDWS Source/mdo/mdo/src/mdo/domain/sm/trash/TiuNoteRecord.java
package gov.va.med.mhv.sm.model;

import java.io.Serializable;
import java.util.Date;

public class TiuNoteRecord extends BaseModel implements Serializable{

	/**
	 * 
	 */
	private static final long serialVersionUID = 615672084864146276L;
	private String vistaDiv;
	private Long threadId; 
	private Long lastMessageId;
	private Date lockedDate;
	private String conversationId;
	private Date noteCreationDate;
	private String comments;
	
	public TiuNoteRecord(){
		noteCreationDate = new Date();
		threadId = 0L;
		lastMessageId = 0L;
	}
	
	public String getVistaDiv() {
		return vistaDiv;
	}
	public void setVistaDiv(String vistaDiv) {
		this.vistaDiv = vistaDiv;
	}
	public Long getThreadId() {
		return threadId;
	}
	public void setThreadId(Long threadId) {
		this.threadId = threadId;
	}
	public Long getLastMessageId() {
		return lastMessageId;
	}
	public void setLastMessageId(Long lastMessageId) {
		this.lastMessageId = lastMessageId;
	}
	public Date getLockedDate() {
		return lockedDate;
	}
	public void setLockedDate(Date lockedDate) {
		this.lockedDate = lockedDate;
	}
	public String getConversationId() {
		return conversationId;
	}
	public void setConversationId(String conversationId) {
		this.conversationId = conversationId;
	}
	public Date getNoteCreationDate() {
		return noteCreationDate;
	}
	public void setNoteCreationDate(Date noteCreationDate) {
		this.noteCreationDate = noteCreationDate;
	}
	public String getComments() {
		return comments;
	}
	public void setComments(String comments) {
		this.comments = comments;
	}

}

File: OtherComponents/MDWSvistalayer/MDWS Source/mdo/mdo/src/mdo/domain/sm/TiuNoteRecord.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace gov.va.medora.mdo.domain.sm
{
    [Serializable]
    public class TiuNoteRecord : BaseModel
    {
        private string _vistaDiv;

        public string VistaDiv
        {
            get { return _vistaDiv; }
            set { _vistaDiv = value; }
        }
        private Int64 _threadId;

        public Int64 ThreadId
        {
            get { return _threadId; }
            set { _threadId = value; }
        }
        private Int64 _lastMessageId;

        public Int64 LastMessageId
        {
            get { return _lastMessageId; }
            set { _lastMessageId = value; }
        }
        private DateTime _lockedDate;

        public DateTime LockedDate
        {
            get { return _lockedDate; }
            set { _lockedDate = value; }
        }
        private string _conversationId;

        public string ConversationId
        {
            get { return _conversationId; }
            set { _conversationId = value; }
        }
        private DateTime _noteCreationDate;

        public DateTime NoteCreationDate
        {
            get { return _noteCreationDate; }
            set { _noteCreationDate = value; }
        }
        private string _comments;

        public string Comments
        {
            get { return _comments; }
            set { _comments = value; }
        }
    }
}
